lib/util/easylogging++.h

```diff
@@ -75,7 +75,7 @@
 #   define _ELPP_OS_FREEBSD 1
 #endif
 // Unix
-#if ((_ELPP_OS_LINUX || _ELPP_OS_MAC || _ELPP_OS_FREEBSD) && (!_ELPP_OS_WINDOWS))
+#if ((_ELPP_OS_LINUX || _ELPP_OS_MAC || _ELPP_OS_FREEBSD || __unix__) && (!_ELPP_OS_WINDOWS))
```

Comment:
the better way is make a upstream pull request. but i think it's ok in practical. it's necessary if you want to build on netbsd, which is the only os to support stellar-core on raspberry pi.